templates/foundation/features.html

```diff
@@ -29,6 +29,15 @@ <h3 class="subheader"><span class="highlight"><%= feature.keyword %>: </span><%=
 
               <p class="subheader">
                 <span class="text"><span class="keyword highlight"><%= step.keyword %></span> <%- step.name %></span>
+
+                <% if(step.arguments && step.arguments.length) { %>
+                  <% _.each(step.arguments, function(argument) { %>
+                    <p class="argument"><pre><%= argument %></pre></p>
```

Comment:
you will have to prettify the _argument_ otherwise it would print as `[object Object]`. You can refer to bootstrap theme [here](https://github.com/gkushang/cucumber-html-reporter/blob/develop/templates/_common/bootstrap.hierarchy/features.html#L126-L156)